src/prog_models/composite_model.py

```diff
@@ -3,7 +3,7 @@
 
 from collections.abc import Iterable
 
-from . import PrognosticsModel
+from prog_models.prognostics_model import PrognosticsModel
```

Comment:
```suggestion
from prog_models import PrognosticsModel
```

This is a good change going from relative to absolute, but we want to import from prog_models without the intermediate method. The same is true for all the others below, as well as Linear and Ensemble models